epub build fails if epub_title=None

The config value for `epub_title` defaults to the value set in `html_title`. In a fresh sphinx project created with `sphinx-quickstart`, there is no `html_title` set, but there is the following comment next to it:

```
# The name for this set of Sphinx documents.  If None, it defaults to           
# "<project> v<release> documentation".                                         
#html_title = None
```

So it even suggests setting it to `None`. When doing so the html build still works but the epub build fails with the following exception:

```
$ make epub
sphinx-build -b epub -d _build/doctrees   . _build/epub
Running Sphinx v1.3.1
WARNING: the config value 'html_title' has type `NoneType', defaults to `unicode.'
loading pickled environment... done
building [mo]: targets for 0 po files that are out of date
building [epub]: targets for 1 source files that are out of date
updating environment: 0 added, 0 changed, 0 removed
looking for now-outdated files... none found
preparing documents... done
writing output... [100%] index                                                                                                                                                                                                                                                                                                        
generating indices... genindex
writing additional pages... search
copying static files... done
copying extra files... done
writing mimetype file...
writing META-INF/container.xml file...
writing content.opf file...

Exception occurred:
  File "/home/gregor/.virtualenvs/tmp-c918d3668920861/lib/python2.7/site-packages/sphinx/builders/epub.py", line 210, in esc
    name = name.replace('&', '&amp;')
AttributeError: 'NoneType' object has no attribute 'replace'
The full traceback has been saved in /tmp/sphinx-err-sZ9Bbv.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
Makefile:112: die Regel für Ziel „epub“ scheiterte
make: *** [epub] Fehler 1
```

I assume that the `epub_title` does not default correctly to the `html_title` when `html_title` should be filled in automatically when it's set to `None`.

--- a/doc/config.rst
+++ b/doc/config.rst
@@ -565,8 +565,7 @@ that use Sphinx's HTMLWriter class.
    The "title" for HTML documentation generated with Sphinx's own templates.
    This is appended to the ``<title>`` tag of individual pages, and used in the
    navigation bar as the "topmost" element.  It defaults to :samp:`'{<project>}
-   v{<revision>} documentation'` (with the values coming from the config
-   values).
+   v{<revision>} documentation'`.
 
 .. confval:: html_short_title
 

--- a/sphinx/config.py
+++ b/sphinx/config.py
@@ -92,7 +92,7 @@ class Config(object):
         html_theme_options = ({}, 'html'),
         html_title = (lambda self: l_('%s %s documentation') %
                       (self.project, self.release),
-                      'html'),
+                      'html', [str]),
         html_short_title = (lambda self: self.html_title, 'html'),
         html_style = (None, 'html', [str]),
         html_logo = (None, 'html', [str]),

--- a/sphinx/quickstart.py
+++ b/sphinx/quickstart.py
@@ -190,9 +190,9 @@ html_theme = 'alabaster'
 # Add any paths that contain custom themes here, relative to this directory.
 #html_theme_path = []
 
-# The name for this set of Sphinx documents.  If None, it defaults to
-# "<project> v<release> documentation".
-#html_title = None
+# The name for this set of Sphinx documents.
+# "<project> v<release> documentation" by default.
+#html_title = u'%(project_str)s v%(release_str)s'
 
 # A shorter title for the navigation bar.  Default is the same as html_title.
 #html_short_title = None

